corehq/apps/geospatial/utils.py

```diff
@@ -15,3 +22,53 @@ def get_geo_user_property(domain):
     except GeoConfig.DoesNotExist:
         config = GeoConfig()
     return config.user_location_property_name
+
+
+def _get_mapping(data_items):
+    return {
+        i['id']: f"{i['lat']} {i['lon']}"
+        for i in data_items
+        if i['lat'] and i['lon']
+    }
+
+
+def process_gps_values_for_cases(domain, cases):
+    location_prop_name = get_geo_case_property(domain)
+    case_mapping = _get_mapping(cases)
+    case_list = CommCareCase.objects.get_cases(list(case_mapping.keys()), domain=domain)
+    case_blocks = []
+    for c in case_list:
+        props = c.case_json
```

Comment:
Is this the properties that are to be updated or all properties on the case currently?

Reply:
We are now using `CaseHelper` to only update the custom case property ([0c00723](https://github.com/dimagi/commcare-hq/pull/33420/commits/0c007232c7a461133dc8fd4ea0c971d273c659e0)).